test-suite/Remove_test.go

```diff
@@ -49,20 +49,22 @@ func doRemove(bt testUtils.BackendT, templatesToPkgsToRemove map[string][]string
 
 		if tmpl != "no-deps" {
 			bt.Subtest(tmpl, func(bt testUtils.BackendT) {
-				bt.Subtest("locked", func(bt testUtils.BackendT) {
```

Comment:
oof, actually there should be a locked an unlocked variant. can you keep this locked subtest? we can add unlocked in another PR, but good to keep this as a TODO

Reply:
I can fix that here if you'd prefer

Reply:
Fixed